### PR TITLE
DISTPG-667 Run in a Docker doc

### PR DESCRIPTION
**Pull request by nastena1606**
Reworked install section to include Docker

modified:   docs/apt.md
	new file:   docs/docker.md
	modified:   docs/installing.md
	modified:   docs/repo-overview.md
	modified:   docs/yum.md
	modified:   mkdocs-base.yml
	new file:   snippets/supported-versions.md
	modified:   variables.yml

### CODE REVIEW

File: docs/docker.md [hunk 1]
```diff
@@ -30,41 +30,52 @@ For more information about using Docker, see the [Docker Docs](https://docs.dock
...
+            ```     
 
-        ```{.bash data-prompt="$"}
-        $ echo "POSTGRES_PASSWORD=secret" > .my-pg.env
-        ``` 
+        2. Start the container:       
 
-    2. Start the container:   
+            ```{.bash data-prompt="$"}
+            $ docker run --rm -it --env-file ./.my-pg.env percona-distribution-postgresql:tag
```
Comment (Naeem-Akhter) [inline docs/docker.md]: Start the container:
docker run --name container-name --env-file ./.my-pg.env -d perconalab/percona-distribution-postgresql:tag
Where:
--container-name is the name you want to assign to your container
--tag is the tag specifying the version you want.

Comment (Naeem-Akhter) [reply docs/docker.md]: I think you missed this change. :)

**Review by Naeem-Akhter — CHANGES_REQUESTED** [review]

File: docs/docker.md (hunk 1 above)
Comment (nastena1606) [reply docs/docker.md]: Indeed. Updated, thank you!

**Review by Naeem-Akhter — APPROVED** [review]
LGTM.

**Pull request merged**